mmv1/templates/terraform/encoders/monitoring_monitored_project.go.erb

```diff
@@ -0,0 +1,20 @@
+<%- # the license inside this block applies to this file
+        # Copyright 2023 Google Inc.
+        # Licensed under the Apache License, Version 2.0 (the "License");
+        # you may not use this file except in compliance with the License.
+        # You may obtain a copy of the License at
+        #
+        #     http://www.apache.org/licenses/LICENSE-2.0
+        #
+        # Unless required by applicable law or agreed to in writing, software
+        # distributed under the License is distributed on an "AS IS" BASIS,
+        # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+        # See the License for the specific language governing permissions and
+        # limitations under the License.
+-%>
+name := d.Get("name").(string)
+name = GetResourceNameFromSelfLink(name)
+metricsScope := d.Get("metrics_scope").(string)
+metricsScope = GetResourceNameFromSelfLink(metricsScope)
+obj["name"] = fmt.Sprintf("locations/global/metricsScopes/%s/projects/%s", metricsScope, name)
```

Comment:
Is this just a strange API quirk that requires the name to be set in this format?

Reply:
Yes